azure-proto-compute/VirtualMachineOperations.cs

```diff
@@ -11,14 +11,25 @@ namespace azure_proto_compute
     /// <summary>
     /// Operations over a single virtual machine
     /// </summary>
-    public class VirtualMachineOperations : ResourceOperationsBase<XVirtualMachine, PhVirtualMachine>, ITaggable<XVirtualMachine, PhVirtualMachine>, IDeletableResource<XVirtualMachine, PhVirtualMachine>
+    public class VirtualMachineOperations : ResourceOperationsBase<VirtualMachine, VirtualMachineData>, ITaggable<VirtualMachine, VirtualMachineData>, IDeletableResource<VirtualMachine, VirtualMachineData>
     {
-        public VirtualMachineOperations(ArmResourceOperations genericOperations) : base(genericOperations) { }
+        internal VirtualMachineOperations(ArmResourceOperations genericOperations)
+            : base(genericOperations)
+        {
+        }
 
-        internal VirtualMachineOperations(ArmClientContext context, ResourceIdentifier id, ArmClientOptions clientOptions) : base(context, id, clientOptions) { }
+        internal VirtualMachineOperations(ArmClientContext context, ResourceIdentifier id, ArmClientOptions clientOptions)
+            : base(context, id, clientOptions)
+        {
+        }
 
         public override ResourceType ResourceType => "Microsoft.Compute/virtualMachines";
 
+        public static VirtualMachineOperations FromGeneric(ArmResourceOperations genericOperations)
```

Comment:
CreateFromGeneric?

Reply:
Following the convention from System.DateTime and other types that have .FromBinary(...), . FromTime(...) etc